src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperSide;
+    private double lowerSide;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double upperSide, double lowerSide, double height) {
+        super(color);
+        this.upperSide = upperSide;
+        this.lowerSide = lowerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round(((upperSide + lowerSide) / 2) * height) * 10 / 10;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead of directly calculating the area in the return statement, create separate variables for the sum of the sides and the multiplication result, then return the area based on those variables for better code readability.